plugins/virtualization/kvm_cpu

```diff
@@ -70,12 +73,12 @@ def find_vm_names(pids):
         cmdline = open("/proc/%s/cmdline" % pid, "r")
         result[pid] = clean_vm_name(re.sub(r"^.*-name\x00([a-zA-Z0-9.-_-]*)\x00\-.*$",r"\1", cmdline.readline()))
     return result
-    
+
 def list_pids():
     ''' Find the pid of kvm processes
     @return a list of pids from running kvm
     '''
-    pid = Popen("pidof qemu-system-x86_64", shell=True, stdout=PIPE)
+    pid = Popen("pidof qemu-kvm qemu-system-x86_64 kvm", shell=True, stdout=PIPE)
```

Comment:
As mentioned above I did not change the original behavior. So I just kept `qemu-system-x86_64` as it was the current behavior. I can add though other architectures as well.